software/rasusa/main.nf

```diff
@@ -19,9 +19,8 @@ process RASUSA {
     }
 
     input:
-    tuple val(meta), path(reads)
+    tuple val(meta), path(reads), val(genome_size)
```

Comment:
Is this to deal with the possibility of having 1 genome per sample?

Reply:
yes, in the bactmap pipeline, this process will receive samples genome size either predefined or estimated by sketch_mash

Reply:
🏋🏽

Reply:
@drpatelh in the meta.yml how do specify that the input should be a tuple 
`tuple val(meta), path(reads), val(genome_size)`
?

Reply:
@drpatelh scratch that, I see that you'd examine the main.nf see what the items in the tuple are and then refer to the meta.yml to figure out the structure of those items